deployments/chef/recipes/auto_instrumentation.rb

```diff
@@ -5,6 +5,8 @@
 with_systemd = node['splunk_otel_collector']['auto_instrumentation_systemd'].to_s.downcase == 'true'
 with_java = node['splunk_otel_collector']['with_auto_instrumentation_sdks'].include?('java')
 with_nodejs = node['splunk_otel_collector']['with_auto_instrumentation_sdks'].include?('nodejs') && with_new_instrumentation
+dotnet_supported = %w(x86_64 amd64).include?(node['cpu']['architecture']) && (node['splunk_otel_collector']['auto_instrumentation_version'] == 'latest' || Gem::Version.new(node['splunk_otel_collector']['auto_instrumentation_version']) >= Gem::Version.new('0.99.0'))
```

Comment:
arm support was just added with the recent https://github.com/signalfx/splunk-otel-dotnet/releases/tag/v1.5.0 release. Should we account for this in this PR?

Reply:
We'll first need to update and release our `splunk-otel-auto-instrumentation` deb/rpm packages to support arm64 before we can add arm64 support in our installers.